tests/ops/api/v1/endpoints/test_privacy_notice_endpoints.py

```diff
@@ -646,6 +646,61 @@ def test_get_privacy_notice(
         assert data["disabled"] == privacy_notice.disabled
         assert data["displayed_in_overlay"] == privacy_notice.displayed_in_overlay
 
+    def test_get_privacy_notice_unescaped(
+        self,
+        api_client: TestClient,
+        generate_auth_header,
+    ):
+        auth_header = generate_auth_header(
+            scopes=[scopes.PRIVACY_NOTICE_READ, scopes.PRIVACY_NOTICE_CREATE]
+        )
+
+        # post a new privacy notice that has some sneaky characters
+        maybe_dangerous_description = "user's description <script />"
+        resp = api_client.post(
+            V1_URL_PREFIX + PRIVACY_NOTICE,
+            headers=auth_header,
+            json=[
+                {
+                    "name": "test privacy notice 1",
+                    "description": maybe_dangerous_description,
+                    "origin": "privacy_notice_template_1",
+                    "regions": [
+                        PrivacyNoticeRegion.eu_be.value,
+                        PrivacyNoticeRegion.us_ca.value,
+                    ],
+                    "consent_mechanism": ConsentMechanism.opt_in.value,
+                    "data_uses": ["advertising"],
+                    "enforcement_level": EnforcementLevel.system_wide.value,
+                }
+            ],
+        )
+        assert resp.status_code == 200
```

Comment:
this test fails here, which is the wrong place where it should be failing! I haven't figured out why it's failing yet—it's complaining about the scope, but from what I can tell it has the right scope...

Reply:
because the test is failing here, I don't actually know if the rest of the test actually does what I want it to, but it should be a reasonable attempt

Reply:
im working on a fix here!